src/gallia/command.py

```diff
@@ -212,13 +213,13 @@ def add_class_parser(self) -> None:
         group = self.parser.add_argument_group("generic gallia arguments")
         group.add_argument(
             "--artifacts-dir",
-            default=os.environ.get("PENRUN_ARTIFACTS"),
+            default=self.config.get("gallia.scan.artifacts_dir"),
```

Comment:
Should we use `this.CATEGORY` instead of hard-coding `scan`?

Reply:
done.